src/SymfonyCache/CleanupCacheTagsListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\SymfonyCache;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Listener that allows to cleanup the cache tags header so it's not exposed
```

Comment:
i would prefer to not repeat "cleanup" but say something like:

"Listener to remove the cache tags header before the response is delivered to the client so it's not exposed to the world."